wrangler.toml

```diff
@@ -1,6 +1,9 @@
 name = "command-query-user"
-main = "src/worker.ts"
-compatibility_date = "2024-01-01"
-node_compat = true
+main = "src/index.ts"
+compatibility_date = "2024-09-24"
+compatibility_flags = [ "nodejs_compat" ]
```

Comment:
What do you need nodejs compatibility for?

Reply:
Without this you cannot use any import from `node` and most packages wouldn't work.